### ISSUE TITLE
Does not run on Ubuntu xenial

### ISSUE THREAD

**Issue by probonopd** [KomodoPlatform/antara-gaming-sdk#152]
`/usr/lib/x86_64-linux-gnu/libstdc++.so.6: version 'GLIBCXX_3.4.26' not found (required by /run/firejail/appimage/.appimage-9116/AppRun)`

Please build on Ubuntu xenial with no newer compilers than what it comes with, so that the AppImage can run on all still-supported Ubuntu releases. Thanks.

Reference:
https://github.com/AppImage/appimage.github.io/pull/1832

**Comment by Milerius** [comment]
Oh i just see the issue now @probonopd , yeah it's fixed now, we build on WSL on 16.04 for any appimage

**Comment by probonopd** [comment]
Looks like the issue persists?

**Comment by Milerius** [comment]
@probonopd This is really strange, your message complain about libstdc++, but usually I got the problem with `GLIBC` not `GLIBCXX`

For sure the program have been compiled on Linux WSL 16.04 but with a C++17 compiler, `clang-8` or `clang-9` with a recent `libstdc++` unfortunately we depend on C++17 for this project and recent libstdc++

**Comment by Milerius** [comment]
https://stackoverflow.com/questions/52663287/glibcxx-3-4-26-not-found

**Comment by Milerius** [comment]
One solution would be to compile with the clang C++ library instead of the gcc one

**Comment by probonopd** [comment]
Yes, please do not use a newer version of libstdc++ than what comes with the oldest still-supported Ubuntu release (currently xenial) by default (especially, do not let a newer one get pulled in from a ppa) or else not all still-supported Ubuntu releases will be able to run your application.
